src/validate-changelog.ts

```diff
@@ -90,6 +92,10 @@ type ValidateChangelogOptions = {
  * header, and that there are no unreleased changes present.
  * @param options.tagPrefix - The prefix used in tags before the version number.
  * @param options.formatter - A custom Markdown formatter to use.
+ * @param options.versionBeforePkgRename - A version string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
+ * @param options.tagPrefixBeforePkgRename - A tag prefix string of the package before being renamed.
+ * An optional, which is required only in case of package renamed.
```

Comment:
```suggestion
 * @param [options.tagPrefix] - The prefix used in tags before the version number.
 * @param [options.formatter] - A custom Markdown formatter to use.
 * @param [options.versionBeforePkgRename] - A version string of the package before being renamed.
 * An optional, which is required only in case of package renamed.
 * @param [options.tagPrefixBeforePkgRename] - A tag prefix string of the package before being renamed.
 * An optional, which is required only in case of package renamed.
```